src/srcom.ts

```diff
@@ -57,6 +58,7 @@ export interface Run {
   category: Category;
   time: Duration;
   submitted: Moment;
+  players: [{rel: string; id: string; uri: string}];
```

Comment:
nit: for our internal `Run` type, i think just an array of IDs would be fine instead of these whole objects